src/util/detect-bugs.js

```diff
@@ -1,23 +1,62 @@
-import TRANSITIONRUN_EVENT_LOOP_BUG from "./detect-bugs/transitionrun-loop.js";
-import UNREGISTERED_TRANSITION_BUG from "./detect-bugs/unregistered-transition.js";
+import detectTransitionRunLoopBug from "./detect-bugs/transitionrun-loop.js";
+import detectUnregisteredTransitionBug from "./detect-bugs/unregistered-transition.js";
 
-/**
- * Data structure for all detected bugs.
- * All bugs start off as true, and once their promises resolve, that is replaced with the actual value
- */
-export const bugs = {
-	TRANSITIONRUN_EVENT_LOOP: true,
-	UNREGISTERED_TRANSITION: true,
-};
+class Bugs {
```

Comment:
NIT: Why don't we `export default` right away?

Reply:
Done in https://github.com/LeaVerou/style-observer/pull/111/commits/0f6215b2beb994ab278f77d1daf882f1cecdf00f.